### PR TITLE
Create db from diagram

### PR DESCRIPTION
**Pull request by Javivy**
## In this milestone we implemented the following functions and enhancements:

  - [x] Created database from a diagram
  - [x] Created patient table
  - [x] Created invoices table
  - [x] Created invoice_items table
  - [x] Created medical_histories table
  - [x] Created treatments table
  - [x] Created join table medical_histories_treatments table for many-to-many relationship between medical_histories table and treatments table

### CODE REVIEW

File: schema_based_on_diagram.sql
Comment (brytebee) [inline schema_based_on_diagram.sql:1]: You have done a great job creating the tables. Now go ahead and add indexes to the foreign keys as required to pass this milestone.

image

**Review by brytebee — COMMENTED** [review]
Hi, @Javivy & @kensteph👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great DB design.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  medical_history_id int,
+  treatment_id int,
+  FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id),
+  FOREIGN KEY (treatment_id) REFERENCES treatments (id),
+);
+
+CREATE INDEX invoice_id_index ON invoice_items(invoice_id);
+CREATE INDEX patient_id_index ON medical_histories(patient_id);
+CREATE INDEX medical_history_id_index ON medical_history_treatment(medical_history_id);
+CREATE INDEX treatment_id_index ON medical_history_treatment(treatment_id);
```
Comment (OyePriscilla) [inline schema_based_on_diagram.sql:52]: - [ ] According to the diagram, there suppose to be six (6) foreign ids altogether. You omitted two (invoices and invoice_items), kindly create indexes for the two as well. Optionally, kindly indicate these two foreign keys in their respective tables as well.

Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Javivy 👏 and @kensteph 👏, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Used CREATE TABLE command to create all tables according to the diagram ✔️
Implement join table for the many-to-many relationships ✔️
Made some required changes ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi Team,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🔥🔥💯💯


## Highlights 🔥🔥🔥🔥
- Followed Gitflow ✅
- No Linters error 🤩🔥💯
- Great working on the functionality 🔥🔥




## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**